site/src/app/[domain]/[language]/not-found-content.tsx

```diff
@@ -0,0 +1,22 @@
+"use client";
+
+import Link from "next/link";
+import { useParams } from "next/navigation";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+export const NotFoundContent = (): React.ReactElement => {
```

Comment:
```suggestion
export function NotFoundContent() {
```

Nit: I'd use the same style for function components everywhere

Reply:
50a8b59fed07b2b5d206057f088fc5b48049ce03